src/main/java/com/mael/ttt/ui/UserInterface.java

```diff
@@ -105,4 +108,12 @@ public boolean replay() {
         print(REPLAY);
         return console.read().equals("y");
     }
+
+    public List<String> getBoardIndexesAsStrings(int size) {
```

Comment:
This comment goes into the same direction as the one before.
It feels like the beginning of feature envy here. Asking the board for some things and then evaluating it in the `UserInterface`. Maybe it's wiser to let board handle all the validations itself?